_scripts/getInstances.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs/promises')
+const invidiousApiUrl = 'https://api.invidious.io/instances.json'
+
+fetch(invidiousApiUrl).then(e => e.json()).then(res => {
+  const data = res.filter((instance) => {
+    return !(instance[0].includes('.onion') ||
+      instance[0].includes('.i2p') ||
+      !instance[1].api ||
+      (!process.env.IS_ELECTRON && !instance[1].cors))
```

Comment:
`process.env.IS_ELECTRON` will always be `undefined` here, in the app we set it with webpack. (you just copy pasted it without checking it didn't you 🙈)

Reply:
Running `yarn run get-instances` yields no error so I didn't notice this 😬